### PR TITLE
Update LanguageSetting.ps1

### PR DESCRIPTION
**Pull request by chidozieononiwu**
- Add function for getting existing package versions.

### CODE REVIEW

File: eng/scripts/Language-Settings.ps1 [hunk 1]
```diff
@@ -179,3 +179,25 @@ function Update-python-CIConfig($pkgs, $ciRepo, $locationInDocRepo, $monikerId=$
 
   Set-Content -Path $pkgJsonLoc -Value $jsonContent
 }
+
+function GetExistingPackageVersions ($PackageName, $GroupId=$null) {
```
Comment (weshaggard) [inline eng/scripts/Language-Settings.ps1:183]: nit: I suggest putting the `{` on its own line for large scopes like a function.

File: eng/scripts/Language-Settings.ps1 [hunk 2]
```diff
@@ -179,3 +179,25 @@ function Update-python-CIConfig($pkgs, $ciRepo, $locationInDocRepo, $monikerId=$
...
+  try
+  {
+    $existingVersion = Invoke-RestMethod -Method GET -Uri "https://pypi.python.org/pypi/${PackageName}/json"
+    return ($existingVersion.releases | Get-Member -MemberType NoteProperty).Name
```
Comment (weshaggard) [inline eng/scripts/Language-Settings.ps1:187]: The Name property has the versions?

Comment (chidozieononiwu) [reply eng/scripts/Language-Settings.ps1:187]: weird, but yes.

File: sdk/template/ci.yml [hunk 3]
```diff
@@ -27,7 +27,7 @@ extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
     ServiceDirectory: template
-    TestPipeline: true
+    TestPipeline: false
```
Comment (weshaggard) [inline sdk/template/ci.yml:30]: I assume you want to revert this before merging this change as you only did it for testing the non-test case.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**